pkgs/shells/bash/4.4.nix

```
@@ -25,7 +25,7 @@ let
 in
 
 stdenv.mkDerivation rec {
-  name = "bash-${optionalString interactive "interactive-"}${version}-p${toString (builtins.length upstreamPatches)}";
+  name = "bash-${optionalString (!interactive) "Noninteractive-"}${version}-p${toString (builtins.length upstreamPatches)}";
```

Maybe lowercase? It was lowercase before.
```suggestion
  name = "bash-${optionalString (!interactive) "noninteractive-"}${version}-p${toString (builtins.length upstreamPatches)}";
```